.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
That said, it still misses the point of allowing the user to specify an override for the ansible-lint testenv (which I think is preferable to proliferation of testenvs based on versioning of dependencies).